src/UI/Button.java

```diff
@@ -0,0 +1,13 @@
+package src.UI;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class Button {
+    public JButton button;
```

Comment:
All variables should be private, and we should create a public (or default) setter and/or getter method if we want others to be able to access it. Only provide a setter if we want anyone to be able to change the value.

This should be applied throughout all classes in this project.